crates/noirc_evaluator/src/ssa_refactor/opt/inlining.rs

```diff
@@ -524,72 +520,6 @@ mod test {
         assert_eq!(inlined.functions.len(), 1);
     }
 
-    #[test]
```

Comment:
This PR assumes that the inliner is applied on a defunctionalized SSA to perform the reachability analysis for the cleanup. This test is no longer possible under this assumption